src/main/java/edu/kit/provideq/toolbox/sat/solvers/GamsSATSolver.java

```diff
@@ -10,12 +10,24 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.UUID;
 
 public class GamsSATSolver extends SATSolver {
     private final File gamsDirectory = new File(System.getProperty("user.dir"), "gams");
     private final File satDirectory = new File(gamsDirectory, "sat");
     private final File workingDirectory = new File(System.getProperty("user.dir"), "jobs");//todo move working directory to config
 
+    private final UUID Id = UUID.randomUUID();
```

Comment:
Can you assign a UUID that will be persistent across application instances? If we use load-balancing and multiple backends at some point, this will yield many erroneous requests.

Also, `id` should be in lowerCamelCase

Reply:
What kind of persistence are you thinking of? Just some file that multiple instances share with coordinated read/writes?
If we have multiple toolboxes running in different environments we'll need some other kind of persistence. That would be more complicated as it would need another authoritative instance which can assign ids that every toolbox would need to communicate with.
For now I think `randomUUID()` is trustworthy enough to avoid duplicates.

Reply:
No, I don't think we should use some complicated uuid authority or a File. Just generate random but fixed UUIDs that are already specified in the code. I don't think there will be any UUID collisions but if we generate UUIDs at runtime, they will change once we restart the application or when we run multiple instances of the application.

An easy alternative to UUIDs would be strings, e.g. the fully-qualified class name of the solver (`edu.kit.provideq.toolbox.sat.solvers.GamsSATSolver` in this case) since these have to be unique anyways

Reply:
I see. I think I'm just going to use the class path then, that's a good idea. The UUID is currently converted to a string on the website anyway.